operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -758,6 +767,9 @@ query_frontend:
     annotations: {}
     labels: {}
 
+  headlessService:
```

Comment:
This is mentioned in the migration docs in `migrating-from-cortex.md`. Can you edit that file too so that it doesn't refer to the headless service?